### PR TITLE
Fixes #28: Missing purgeAllContent()

### LINKED ISSUE #28: EzSystems\PlatformHttpCacheBundle\Proxy\TagAwareStore::purgeAllContent() not implemented
purgeAllContent method doesn't seem to be implemented.

https://github.com/ezsystems/ezplatform-http-cache/blob/250149c9a29b7e556c05931a503280568227de87/src/Proxy/TagAwareStore.php#L142

### PR DESCRIPTION
**Pull request by andrerom**
Attempt to fix #28

### CODE REVIEW

File: src/Proxy/TagAwareStore.php [hunk 1]
```diff
@@ -162,6 +162,14 @@ function ($id) {return 'location-' . $id;},
     }
 
     /**
+     * Purge all cache.
+     */
+    protected function purgeAllContent()
+    {
+        $this->fs->remove((new Finder())->in($this->root));
```
[the remark this answers is not in the document]
Comment (andrerom) [reply src/Proxy/TagAwareStore.php]: Right, I see the setter is not used indeed.

Does legacy bridge bark at the method added being protected? Do you have a stack trace?

[the remark this answers is not in the document]
Comment (andrerom) [reply src/Proxy/TagAwareStore.php]: Adjusted in 2e4dc81, you confirm this working as expected with the change you proposed?

[the remark this answers is not in the document]
Comment (volkanakb) [reply src/Proxy/TagAwareStore.php]: thx, works fine

File: src/PurgeClient/LocalPurgeClient.php [hunk 2]
```diff
@@ -55,6 +55,8 @@ function ($tag) {
      */
     public function purgeAll()
     {
-        $this->cacheStore->purgeAllContent();
+        $purgeRequest = Request::create('http://localhost/', 'PURGE');
```
Comment (alongosz) [inline src/PurgeClient/LocalPurgeClient.php]: Hmm, doesn't actual host come from some parameter? At first glance I thought it was a test.

File: src/PurgeClient/LocalPurgeClient.php [hunk 3]
```diff
@@ -55,6 +55,8 @@ function ($tag) {
...
+        $purgeRequest->headers->set('X-Location-Id', '*');
```
Comment (alongosz) [inline src/PurgeClient/LocalPurgeClient.php]: Didn't we have constant for `X-Location-Id` header name? I vaguely remember some discussion about it in one of the other PRs.

File: src/PurgeClient/LocalPurgeClient.php (hunk 2 above)
Comment (andrerom) [reply src/PurgeClient/LocalPurgeClient.php]: This is same as in purge() above, which comes from https://github.com/ezsystems/ezpublish-kernel/blob/1bdad65810d562c8a07be128f1eeb79ec1a5cfa3/eZ/Publish/Core/MVC/Symfony/Cache/Http/LocalPurgeClient.php#L47

reason is host does not matter with the symfony proxy implementation, it's local in the same code base.

File: src/PurgeClient/LocalPurgeClient.php (hunk 3 above)
Comment (andrerom) [reply src/PurgeClient/LocalPurgeClient.php]: yeah, but it's not really relevant class to take constant imho, as it has a special purpose.

**Pull request merged**